Regular Expressions/RegExp.sublime-syntax

```diff
@@ -25,7 +25,7 @@ contexts:
     - include: escaped_char
     - include: charset
     - include: group
-    - match: '([?*+][?+]?)|\{\d*,\d*\}'
+    - match: '([?*+][?+]?)|\{\d*,?\d*\}'
```

Comment:
A regex `{}` just means literal `{}` and so does `{,}`.

May use this one.
`- match: '([?*+][?+]?)|\{,\d+\}|\{\d+,?\d*\}'`

![image](https://cloud.githubusercontent.com/assets/6594915/14522886/6fe780ee-0263-11e6-8c75-9d95a22a52d2.png)